src/templates/join-page.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+import Layout from '../components/Layout'
+import AlternatingContent from '../components/AlternatingContent'
+import contentBlockPropTypes from '../components/TwoColumnContentBlock/contentBlockPropTypes'
+import OurPeople from '../components/People/OurPeople'
+
+export const JoinPageTemplate = ({
+  contentItems,
+  trackleadsDescription,
+  ourPeopleTitle,
+}) => (
+  <>
+    <div className="section">
```

Comment:
This is an extra div that's not needed AFAIK, just <><Alternating>, <OurPeople></>. Both already define their own section containers. As an example, look at index-page.js